eng/mark-shipped.ps1

```diff
@@ -0,0 +1,51 @@
+[CmdletBinding(PositionalBinding=$false)]
+param ()
+
+Set-StrictMode -version 2.0
+$ErrorActionPreference = "Stop"
+
+function MarkShipped([string]$dir) {
```

Comment:
functions should follow approvedverb-noun rule, so more likely Set-Shipped

parameters should be capitalized and ideally whole word. $Directory, you might want to add [Parameter(Mandatory)] to force getting the value.